mne/channels/layout.py

```diff
@@ -1000,6 +1000,80 @@ def _merge_grad_data(data, method='rms'):
     return data.reshape(data.shape[:1] + orig_shape[1:])
 
 
+def _combine_meg_grads(inst, method='rms'):
+    """Combine pairs of gradiometer channels."""
+    # TODO:
+    # - modify coil_type and kind to a combined gradiometer
```

Comment:
kind should still be MEG I think. For coil_type I think we'd maybe want to add a 3016 here:

https://github.com/mne-tools/mne-python/blob/main/mne/io/constants.py#L929

We'll need to make a PR to fif-constants for this. We can do this later, but to make `mne/io/tests/test_constants.py` happy you can for fif-constants, make a new branch with a commit that adds it, push to your fork, and tell test_constants to use your commit like here:

https://github.com/mne-tools/mne-python/pull/11064/files#diff-d545e84fd98e16f320b83c21e8c35a67ed64d378ded8fb35d3734e9f2196b53eR23-R25

Reply:
Thanks - I think this is done in [5fa25d1](https://github.com/mne-tools/mne-python/pull/11082/commits/5fa25d12a7e2978b199ca0ca929b7c643a8fe2d6) and [1e87305](https://github.com/mne-tools/mne-python/pull/11082/commits/1e87305c1f40a744786f009727e9c8aaa3aeb5a5) referencing this fork of fiff-constants https://github.com/AJQuinn/fiff-constants/tree/grad_combined

Not sure how critical the naming choice is?